### PR TITLE
Add the option to copy the reminder to the clipboard

### PR DESCRIPTION
**Pull request by farzadmf**
Didn't add a test because it's not _technically_ part of the answer :slightly_smiling_face:

### CODE REVIEW

File: survey/asker.go [hunk 1]
```diff
@@ -173,13 +175,29 @@ func Ask() error {
...
 
 	err := survey.Ask(qs, &answer)
 	if err != nil {
 		return err
 	}
 
-	fmt.Println(answer)
+	if answer.Clipboard {
+		if err := clipboard.WriteAll(answer.String()); err != nil {
+			log.Fatal(err)
```
Comment (skanehira) [inline survey/asker.go]: There have to return `err`

**Review by skanehira — CHANGES_REQUESTED** [review]
Thanks 👍 
I have left a comment.

File: survey/asker.go (hunk 1 above)
Comment (farzadmf) [reply survey/asker.go]: Oh wow, sorry, I missed that :stuck_out_tongue: will fix right away

**Comment by farzadmf** [comment]
Fixed the issue @skanehira please check again :slightly_smiling_face: 

Also, how do we create a new release?

**Comment by skanehira** [comment]
LGTM 👍 

> Also, how do we create a new release?

When I push new git tag, then GitHub Actions will create new release :)
Please wait a lille, I'll do it.

**Pull request merged**